contributing/getting-started-e2e.md

```diff
@@ -171,10 +169,10 @@ amphtml uses Node.js, the yarn package manager and the Gulp build system to buil
 
 * The AMP Project uses Gulp as our build system.   Gulp uses a configuration file ([gulpfile.js](https://github.com/ampproject/amphtml/blob/master/gulpfile.js)) to build amphtml (including the amphtml javascript) and to start up the Node.js server with the proper settings.  You don't really have to understand exactly what it is doing at this point--you just have to install it and use it.
 
-   You can install Gulp using yarn:
+   You can install Gulp using npm:
 
    ```
-   yarn global add gulp
+   sudo npm install -g gulp
```

Comment:
Is `sudo` required on all platforms? If not, don't include it here.

Reply:
Removed `sudo` from the command and added a comment below to clarify